app/services/users.js

```diff
@@ -29,16 +29,15 @@ const getUserByEmail = async email => {
 
 const getUsers = async (limit = DEFAULT_LIMIT, offset = DEFAULT_OFFSET) => {
   try {
-    logger.info(`usersController::getUsers::offset::${offset}`);
-    logger.info(`usersController::getUsers::limit::${limit}`);
+    logger.info(`usersController::getUsers::limit::${limit}::offset::${offset}`);
     const response = await User.findAndCountAll({
       offset,
       limit,
       attributes: ['id', 'firstName', 'lastName', 'email']
     });
     return response;
   } catch (error) {
-    logger.error(error);
+    logger.error(`users-service::getUsers::error::${error}`);
```

Comment:
What I mean is that this `error` variable does not have an implicit toString, so it should not be printed because it will show [Object object]